codespell_lib/data/dictionary.txt

```diff
@@ -935,6 +935,7 @@ archtypes->archetypes
 archvie->archive
 archvies->archives
 arcticle->article
+Ardiuno->Arduino
```

Comment:
I think this will need to be `arduino` because caps aren't handled properly.

@luzpaz maybe we should add a `Travis` check that all `entry == entry.lower()`?

Reply:
I did wonder, but then found a few US states with capitals. Will they not work, or only match if upper case, or?

Reply:
They won't work. This is fixed in #482